diff -urN radius-1.5.org/include/radiusd.h radius-1.5/include/radiusd.h
--- radius-1.5.org/include/radiusd.h	2007-06-27 14:54:21.000000000 +0200
+++ radius-1.5/include/radiusd.h	2007-07-01 21:27:18.441509896 +0200
@@ -340,7 +340,6 @@
 extern int max_threads;
 extern int num_threads;
 #ifdef USE_SERVER_GUILE
-extern unsigned scheme_gc_interval;
 extern u_int scheme_task_timeout;
 #endif
 #ifdef USE_SNMP
