--- radius-1.1/radscm/rscm_utmp.c.orig	2003-06-19 14:39:42.000000000 +0200
+++ radius-1.1/radscm/rscm_utmp.c	2003-10-21 21:51:19.461832888 +0200
@@ -44,7 +44,7 @@
             SCM LIST,
             SCM RADUTMP_FILE,
             SCM RADWTMP_FILE),
-"Write the supplied data into the radutmp file. If RADWTMP_FILE is not nil
+"Write the supplied data into the radutmp file. If RADWTMP_FILE is not nil\n\
 the constructed entry is also appended to WTMP_FILE.")
 #define FUNC_NAME s_rad_utmp_putent
 {
